fixtures/ios_app_with_tests/Project.swift

```diff
@@ -55,6 +55,7 @@ let project = Project(
             platform: .macOS,
             product: .framework,
             bundleId: "io.tuist.MacFramework",
+            deploymentTarget: .macOS(targetVersion: "10.15"),
```

Comment:
was this needed for a particular lint check?

Reply:
Not really. It's my attempt to fix some acceptance tests that are failing. It errors because the minimum deployment target set by Xcode by default is higher than the os in which the acceptance tests run.

Reply:
Xcode 12.3 sets the minimum deployment target to to 11.x when no minimum deployment target is specified. Since CI runs in macOS 10.15, xcodebuild fails trying to run the tests.